src/pages/candidate-details/candidate-details.selector.ts

```diff
@@ -109,3 +109,6 @@ export const getDetails = (slot: any): Details => {
   };
   return details;
 };
+export const processSpecialNeeds = (slot: any): string | string[] => {
```

Comment:
Could we handle `None` as a first element of `string[]`? Because in that scenario we wouldn't need to have more than one type for special needs.

Reply:
Yeah I think we could, but we need to do a check to see if SN is populated, so we can format and set the 'no special needs' content and show the indicator. We could check on the first element but I feel that almost the same thing as what I'm doing.